ocs_ci/ocs/resources/pod.py

```diff
@@ -192,6 +195,25 @@ def get_mount_path(self):
             ).get('containers')[0].get('volumeMounts')[0].get('mountPath')
         )
 
+    def workload_setup(self, storage_type, jobs=1):
```

Comment:
This change is made to provide a way to set up the environment for running IO (like, installing fio) in pod so that 'run_io' method can instantly start IO when called.
Otherwise, after calling 'run_io', it will take 25-30 seconds to actually start IO.
This is most helpful in cases like running other operations in parallel with IO because other operations don't have to wait for 25-30 seconds for IO to start.
Also, ' wl_setup_done' attribute is used to ensure that 'setup'  method is  called only first time in cases where IO has to run on pod for more than one time.